gen/ad_user_mu_ucn

```diff
@@ -52,23 +65,35 @@ open FILE,">:encoding(UTF-8)","$baseDnFileName" or die "Cannot open $baseDnFileN
 print FILE $baseDN;
 close(FILE);
 
+#my %allChipNumbers;  #used for consistency check - each chip number should belong to only one user
```

Comment:
I'm not sure, what are the constraints in AD, but having this check implemented would be generally better, so either implement it or remove the unused comment.